src/localization/en.js

```diff
@@ -93,4 +93,7 @@ export default {
   pocketButtonEditWeightText: 'Edit weight ',
   information: 'Information ',
   collectionPoint: 'Collection point',
+
+  // Map
+  endTravel: 'End travel',
```

Comment:
This string feels wrong to me. Id like to see something like "complete trip", or "complete yout travel/journey".